services/src/main/scala/cromwell/pipeline/service/ProjectVersioning.scala

```diff
@@ -3,6 +3,8 @@ package cromwell.pipeline.service
 import java.nio.file.Path
 
 import cromwell.pipeline.datastorage.dto.{ Project, ProjectFile, Version }
+import cromwell.pipeline.datastorage.dto.Project
+import cromwell.pipeline.datastorage.dto.{ ProjectFile, Version }
```

Comment:
Looks like you missed that import. It should be fixed too.

Reply:
Fixed